src/Idle.DataAccess/Fields/Frameworks/Flutter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Idle.DataAccess.Buffs;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Idle.DataAccess.Fields.Frameworks
+{
+    /// <summary>
+    /// Class for the flutter framework
+    /// </summary>
+    public class Flutter : FrameworkBase
```

Comment:
`FrameworkBase` implements an abstract property `PlayerLevel` but it is not implemented in the class `Flutter`.
Always rebuild the application and execute all tests (none are there yet) before you make a PR.
The solution should be in a bug-free and buildable state when doing a PR. 
So we dont merge any issues into the target branch.

Reply:
This makes sense, should've realized this earlier.